src/Uploadable/Event/UploadableBaseEventArgs.php

```diff
@@ -104,6 +105,16 @@ public function getEntityManager()
         return $this->em;
```

Comment:
I decided against firing a deprecation here as an EntityManagerInterface is indeed injected